include/timer.h

```diff
@@ -1,5 +1,23 @@
 #ifndef _TIMER_H
 #define _TIMER_H
+#include "common.h"
 
+#define MAX_TIMER   128
+
+#define UNUSE   0
+#define USED    1
+
+typedef struct TIMER {
+    struct TIMER *next;
+    unsigned int timeout, flags;
+    int data;
+}timer;
+
+typedef struct TIMER_POOL {
+    unsigned int count, next;
+    struct TIMER *current;
+    struct TIMER pool[MAX_TIMER];
+}timer_zone;
```

Comment:
That's actually a matter of taste, but I usually stick to the occam's razor rule, - be minimalistic in the code. We don't actually need those singletons like `timer_zone` and `timer`. And declaring the `timer_zone timers;` in the timer.c:5 can be immediatelly like TIMER_POOL timer.
And besides, I don't remember about how it was in our convention, but usually, people use the UPPERCASE_NAMES for constants and all magics. Naming the structures are usually lower_case_names.
@wjn740 @crazyss what do you think.

Reply:
No problem. I think you are right. I don't have too much experience on this, but I'd like to improve it.